README.md

```diff
@@ -379,6 +379,42 @@ storage:
       overwrite: true
 ```
 
+#### Ollama
+
+The ollama sysext can be configured by using the following snippet:
+
+```
+variant: flatcar
+version: 1.0.0
+storage:
+  files:
+    - path: /opt/extensions/ollama/ollama-0.3.6-x86-64.raw
+      contents:
+        source: https://github.com/flatcar/sysext-bakery/releases/download/latest/ollama-0.3.6-x86-64.raw
+    - path: /etc/sysupdate.ollama.d/ollama.conf
+      contents:
+        source: https://github.com/flatcar/sysext-bakery/releases/download/latest/ollama.conf
+  links:
+    - target: /opt/extensions/ollama/ollama-0.3.6-x86-64.raw
+      path: /etc/extensions/ollama.raw
+      hard: false
+systemd:
+  units:
+    - name: ollama.service
+      enabled: true
+      dropins:
+        - name: 10-ollama-env-override.conf
+          contents: |
+            [Service]
+            Environment=OLLAMA_MODELS="/var/lib/ollama"
+            Environment=HOME="/var/lib/ollama"
+            Environment=OLLAMA_RUNNERS_DIR="/var/lib/ollama"
```

Comment:
do the defaults of ollama in the absence of this file make sense? can we include these by default in the sysext

Reply:
Great question, I am definitely open to including these as the defaults configured in the sysext itself, but the reason I was hesitant to do that initially is because you likely want a fairly sizable volume at this location given that each [model](https://ollama.com/library) you'll be pulling down is going vary greatly in size with the smallest ones being 500mb and largest ones being 75+ GB. 

I didn't want to impose a requirement to have the volume mounted at a specific location if folks have different preferences for that, but the sysext could as long as it's documented.

What do you think?

Reply:
by default `/var` is on the same partition as `/`, which is the last partition on the boot disk and gets expanded to the available disk size. So that's a good default.

Users can always override the environment variables in their own systemd drop-in file with these variables.